spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/StringToDurationConverter.java

```diff
@@ -28,6 +28,7 @@
  * @author Andy Wilkinson
  * @since 2.0.0
  */
+// FIXME Andy - what is the solution to this again? See micrometer #160
```

Comment:
I think we can just get rid of it in Boot 2.0. The formatting conversion service can do Duration.parse(String)

Reply:
Deleted.